firefox-ios/Client/Frontend/Browser/Tabs/Views/CustomSelectorView/TabTraySelectorView.swift

```diff
@@ -133,22 +135,6 @@ final class TabTraySelectorView: UIView,
         selectNewSection(newIndex: indexPath.item)
     }
 
-    // MARK: - Scroll Snap
```

Comment:
@OrlaM I removed this since we don't scroll anymore on the collection view and this was breaking VoiceOver swipe navigation.